src/exception.cr

```diff
@@ -2,18 +2,39 @@ require "callstack"
 
 CallStack.skip(__FILE__)
 
+# Represents errors that occur during application execution.
+#
+# Descendants of class Exception are used to communicate between raise and
```

Comment:
But it *is* used everywhere: https://github.com/crystal-lang/crystal/blob/master/src/raise.cr#L189.